### PR TITLE
fix: disable dynamic translations on linux

### PR DESCRIPTION
**Pull request by richard-ramos**

### CODE REVIEW

File: ui/app/AppLayouts/Profile/Sections/LanguageModal.qml [hunk 1]
```diff
@@ -22,6 +22,26 @@ ModalPopup {
...
+            id: languageChangeConfirmationDialog
+            property string newLocale
+
+            title: qsTr("Change language")
+            confirmationText: qsTr("Display language has been changed. You must restart the application for changes to take effect.")
+            showCancelButton: true
+            confirmButtonLabel: qsTr("Close the app now")
+            cancelButtonLabel: qsTr("I'll do that later")
+            onConfirmButtonClicked: {
+                console.log("NEW LOCALE?", newLocale)
```
Comment (richard-ramos) [inline ui/app/AppLayouts/Profile/Sections/LanguageModal.qml:35]:
```suggestion
```

**Review by jrainville — APPROVED** [review]

**Review by pinebit — APPROVED** [review]

File: ui/app/AppLayouts/Profile/Sections/LanguageModal.qml [hunk 2]
```diff
@@ -22,6 +22,25 @@ ModalPopup {
             id: languageGroup
         }
 
+        ConfirmationDialog {
```
Comment (iurimatias) [inline ui/app/AppLayouts/Profile/Sections/LanguageModal.qml:25]: I think there is a pragma or something that makes this only appear in linux

**Review by iurimatias — CHANGES_REQUESTED** [review]

File: ui/app/AppLayouts/Profile/Sections/LanguageModal.qml (hunk 2 above)
Comment (borismelnik) [reply ui/app/AppLayouts/Profile/Sections/LanguageModal.qml:25]: `visible: Qt.platform.os === "linux"` or from Loader (more performance)

**Review by borismelnik — APPROVED** [review]

**Comment by richard-ramos** [comment]
Depends on
- https://github.com/status-im/dotherside/pull/33
- https://github.com/status-im/nimqml/pull/24

**Pull request merged**